firebase-database/firebase-database.gradle

```diff
@@ -84,14 +84,14 @@ dependencies {
     androidTestImplementation "androidx.annotation:annotation:1.1.0"
     androidTestImplementation 'androidx.test:rules:1.2.0'
     androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation "com.google.truth:truth:$googleTruthVersion"
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-core:2.9.8'
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
     androidTestImplementation 'junit:junit:4.12'
+    androidTestImplementation 'org.hamcrest:hamcrest:2.2'
+    androidTestImplementation 'org.hamcrest:hamcrest-library:2.2'
     androidTestImplementation 'net.java:quickcheck:0.6'
-    androidTestAnnotationProcessor 'net.java:quickcheck-src-generator:0.6'
```

Comment:
These annotations were not used and caused conflicts because their maximum version is 1.6